src/components/organisms/History.tsx

```diff
@@ -71,39 +71,55 @@ export default function History() {
       },
     };
 
+    // TODO
     const res = await fetch(
       `${url}/device-data/${device_id}/historical/week`,
       options
     );
     const data = await res.json();
-
     console.log("data: ", data);
+
```

Comment:
Change `TODO` to `TODO Implement catch`.
Replace `fetch` to `axios`.

Reply:
```ts
const res = await axios.get(${url}/device-data/${device_id}/historical/week`)
const data = res.data;
```

Also delete the `options` too.

Reply:
Done